resources/api/resource.py

```diff
@@ -113,6 +116,12 @@ def get_user_permissions(self, obj):
     def get_required_reservation_extra_fields(self, obj):
         return REQUIRED_RESERVATION_EXTRA_FIELDS if obj.need_manual_confirmation else []
 
+    def get_is_favorite(self, obj):
+        request = self.context.get('request', None)
+        if not (request and request.user.is_authenticated()):
+            return False
+        return request.user.favorite_resources.filter(id=obj.id).exists()
```

Comment:
Yeah good point, I think this can be easily solved by prefetching the favorite resources, adding that.